terasoluna-gfw-functionaltest-web/src/main/java/org/terasoluna/gfw/functionaltest/app/el/ElController.java

```diff
@@ -106,7 +106,8 @@ public String urlLinkString() {
         return "el/linkOutput";
     }
 
-    @RequestMapping(value = "5_4", method = RequestMethod.GET)
+    @RequestMapping(value = "5_4", method = { RequestMethod.GET,
+            RequestMethod.POST })
```

Comment:
- [ ] Why you need to treat `POST` request additionally?

Reply:
I found it was an unnecessary fix, so I put it back.
c282b497f5db5b9d7b2f0901b69ec5217e1c4224